### ISSUE TITLE
Lint Fails

### ISSUE THREAD

**Issue by kreig303** [issc29/probot-prevent-public-repos#8]
Platform:

macOS Mojave
Node 10.14

Steps to reproduce:
```
$ git clone ...
$ npm it
```
Output:
```
npm WARN deprecated github@10.1.0: 'github' has been renamed to '@octokit/rest' (https:/git.io/vNB11)
npm WARN deprecated circular-json@0.3.3: CircularJSON is in maintenance only, flatted is its successor.

> dtrace-provider@0.8.7 install /probot-prevent-public-repos/node_modules/dtrace-provider
> node-gyp rebuild || node suppress-error.js

  ACTION binding_gyp_ndtp_target_build_ndtp .
  TOUCH Release/obj.target/ndtp.stamp

> fsevents@1.2.4 install /probot-prevent-public-repos/node_modules/fsevents
> node install

[fsevents] Success: "/probot-prevent-public-repos/node_modules/fsevents/lib/binding/Release/node-v64-darwin-x64/fse.node" already installed
Pass --update-binary to reinstall or --build-from-source to recompile
npm notice created a lockfile as package-lock.json. You should commit this file.
added 828 packages from 491 contributors and audited 28093 packages in 20.887s
found 1 high severity vulnerability
  run `npm audit fix` to fix them, or `npm audit` for details

> prevent-public-repos@1.0.0 test /probot-prevent-public-repos
> jest && standard

 PASS  test/PreventPublicRepos.test.js
  ● Console

    console.info lib/PreventPublicRepos.js:56
      Repo: test-pro was publicized but enablePrivateToPublic is set to false
    console.info lib/PreventPublicRepos.js:46
      Repo: test-pro is public
    console.info lib/PreventPublicRepos.js:46
      Repo: test-pro is public
    console.info lib/PreventPublicRepos.js:46
      Repo: test-pro is public
    console.info lib/PreventPublicRepos.js:64
      Repo: test-pro is part of the exclusion list
    console.info lib/PreventPublicRepos.js:46
      Repo: test-pro is public
    console.info lib/PreventPublicRepos.js:72
      Repo: test-pro visibility was created private
    console.info lib/PreventPublicRepos.js:46
      Repo: test-pro is public
    console.info lib/PreventPublicRepos.js:46
      Repo: test-pro is public
    console.info lib/PreventPublicRepos.js:79
      Privatizing Repo: test-pro
    console.info lib/PreventPublicRepos.js:79
      Privatizing Repo: test-pro

 PASS  test/index.test.js

Test Suites: 2 passed, 2 total
Tests:       18 passed, 18 total
Snapshots:   0 total
Time:        1.128s
Ran all test suites.
standard: Use JavaScript Standard Style (https:/standardjs.com)
  /probot-prevent-public-repos/test/PreventPublicRepos.test.js:21:5: 'payloadPublicizied' is not defined.
  /probot-prevent-public-repos/test/PreventPublicRepos.test.js:30:5: 'payloadCreatedPublic' is not defined.
  /probot-prevent-public-repos/test/PreventPublicRepos.test.js:39:5: 'payloadCreatedPrivate' is not defined.
  /probot-prevent-public-repos/test/PreventPublicRepos.test.js:52:7: 'spyExecutePrivatize' is not defined.
  /probot-prevent-public-repos/test/PreventPublicRepos.test.js:53:7: 'spyMonitorOnly' is not defined.
  /probot-prevent-public-repos/test/PreventPublicRepos.test.js:56:7: 'spyExecutePrivatize' is not defined.
  /probot-prevent-public-repos/test/PreventPublicRepos.test.js:57:7: 'spyMonitorOnly' is not defined.
  /probot-prevent-public-repos/test/PreventPublicRepos.test.js:61:32: 'payloadPublicizied' is not defined.
  /probot-prevent-public-repos/test/PreventPublicRepos.test.js:65:14: 'spyExecutePrivatize' is not defined.
  /probot-prevent-public-repos/test/PreventPublicRepos.test.js:66:14: 'spyMonitorOnly' is not defined.
  /probot-prevent-public-repos/test/PreventPublicRepos.test.js:70:32: 'payloadPublicizied' is not defined.
  /probot-prevent-public-repos/test/PreventPublicRepos.test.js:75:14: 'spyExecutePrivatize' is not defined.
  /probot-prevent-public-repos/test/PreventPublicRepos.test.js:76:14: 'spyMonitorOnly' is not defined.
  /probot-prevent-public-repos/test/PreventPublicRepos.test.js:80:32: 'payloadCreatedPublic' is not defined.
  /probot-prevent-public-repos/test/PreventPublicRepos.test.js:85:14: 'spyExecutePrivatize' is not defined.
  /probot-prevent-public-repos/test/PreventPublicRepos.test.js:86:14: 'spyMonitorOnly' is not defined.
  /probot-prevent-public-repos/test/PreventPublicRepos.test.js:90:32: 'payloadCreatedPublic' is not defined.
  /probot-prevent-public-repos/test/PreventPublicRepos.test.js:95:14: 'spyExecutePrivatize' is not defined.
  /probot-prevent-public-repos/test/PreventPublicRepos.test.js:96:14: 'spyMonitorOnly' is not defined.
  /probot-prevent-public-repos/test/PreventPublicRepos.test.js:100:32: 'payloadCreatedPublic' is not defined.
  /probot-prevent-public-repos/test/PreventPublicRepos.test.js:104:14: 'spyExecutePrivatize' is not defined.
  /probot-prevent-public-repos/test/PreventPublicRepos.test.js:105:14: 'spyMonitorOnly' is not defined.
  /probot-prevent-public-repos/test/PreventPublicRepos.test.js:109:32: 'payloadCreatedPublic' is not defined.
  /probot-prevent-public-repos/test/PreventPublicRepos.test.js:114:14: 'spyExecutePrivatize' is not defined.
  /probot-prevent-public-repos/test/PreventPublicRepos.test.js:115:14: 'spyMonitorOnly' is not defined.
  /probot-prevent-public-repos/test/PreventPublicRepos.test.js:119:32: 'payloadCreatedPrivate' is not defined.
  /probot-prevent-public-repos/test/PreventPublicRepos.test.js:121:14: 'spyExecutePrivatize' is not defined.
  /probot-prevent-public-repos/test/PreventPublicRepos.test.js:122:14: 'spyMonitorOnly' is not defined.
  /probot-prevent-public-repos/test/PreventPublicRepos.test.js:126:32: 'payloadCreatedPublic' is not defined.
  /probot-prevent-public-repos/test/PreventPublicRepos.test.js:131:14: 'spyExecutePrivatize' is not defined.
  /probot-prevent-public-repos/test/PreventPublicRepos.test.js:132:14: 'spyMonitorOnly' is not defined.
  /probot-prevent-public-repos/test/PreventPublicRepos.test.js:136:32: 'payloadCreatedPublic' is not defined.
  /probot-prevent-public-repos/test/PreventPublicRepos.test.js:140:14: 'spyExecutePrivatize' is not defined.
  /probot-prevent-public-repos/test/PreventPublicRepos.test.js:141:14: 'spyMonitorOnly' is not defined.
  /probot-prevent-public-repos/test/PreventPublicRepos.test.js:147:32: 'payloadCreatedPublic' is not defined.
  /probot-prevent-public-repos/test/PreventPublicRepos.test.js:155:32: 'payloadCreatedPublic' is not defined.
  /probot-prevent-public-repos/test/PreventPublicRepos.test.js:163:32: 'payloadCreatedPublic' is not defined.
  /probot-prevent-public-repos/test/PreventPublicRepos.test.js:176:32: 'payloadCreatedPublic' is not defined.
  /probot-prevent-public-repos/test/PreventPublicRepos.test.js:191:32: 'payloadCreatedPublic' is not defined.
  /probot-prevent-public-repos/test/PreventPublicRepos.test.js:210:32: 'payloadCreatedPublic' is not defined.
npm ERR! Test failed.  See above for more details.
```

**Comment by kreig303** [comment]
This has been addressed in #10

**Comment by issc29** [comment]
Fixed in #10
